code/game/gamemodes/blob/powers.dm

```diff
@@ -275,9 +275,13 @@
 		return
 	last_attack = world.time
 	OB.expand(T, 0, blob_reagent_datum.color)
-	for(var/mob/living/L in T)
-		blob_reagent_datum.reaction_mob(L, TOUCH, 25)
-		blob_reagent_datum.send_message(L)
+	mob_loop:
+		for(var/mob/living/L in T)
+			for(var/F in L.faction)
+				if(F == "blob") // no friendly fire
```

Comment:
why not just if("blob" in L.faction)?
then you wouldn't need a named loop; there'd only be the one